.github/container/test-axlearn.sh

```diff
@@ -193,13 +193,24 @@ TEST_8_DEVICES_FILES=("gda_test.py"
 )
 TEST_8_DEVICES_WITH_PATHS=()
 for file in "${TEST_8_DEVICES_FILES[@]}"; do
-    found_files=$(find . -name "$file" -type f 2>/dev/null)
-    if [[ -n "$found_files" ]]; then
-        while IFS= read -r found_file; do
+    # Handle the ambiguous 'utils_test.py' as a special case.
```

Comment:
This seems a bit complicated. Why not just list out the paths? It looks like you are just putting `axlearn/common` in front of them all.
```
% find . -name utils_test.py -or -name trainer_test.py -or -name input_dispatch_test.py -or -name input_base_test.py -or -name gda_test.py
./axlearn/cli/utils_test.py
./axlearn/common/utils_test.py
./axlearn/common/trainer_test.py
./axlearn/common/input_dispatch_test.py
./axlearn/common/quantized_dot_general/utils_test.py
./axlearn/common/input_base_test.py
./axlearn/common/flash_attention/utils_test.py
./axlearn/common/gda_test.py
./axlearn/cloud/gcp/utils_test.py
./axlearn/cloud/gcp/runners/utils_test.py
./axlearn/cloud/common/utils_test.py
```